FreifunkMemorandumofUnderstanding_it.md

```diff
@@ -0,0 +1,67 @@
+* ["Wir speichern nicht!" (Non conserviamo i log)](http://www.wirspeichernnicht.de/)
+* l'[etica hacker del Chaos Computer Club (CCC)](http://www.ccc.de/de/hackerethik), che è trasferibile alle reti libere
+* la dichiarazione di rifiuto della discriminazione e del fascismo del CCC, ["Farbe erklären gegen Rechts"](http://www.ccc.de/updates/2005/unvereinbarkeitserklaerung), in particolare la sezione "Die Erklärung"
+
+### Principi tecnici
+
+Il Pico Peering Agreement è alla base delle nostre reti. Per noi i pilastri che costituiscono una rete libera sono i seguenti:
+
+* I nostri nodi formano una rete mesh interconnettendosi gli uni con gli altri. Il termine rete Freifunk si riferisce a questa rete mesh. Finisce dove inizia Internet (ad esempio nel momento in cui un router instrada il traffico dati verso Internet), e finisce dove comincia la rete privata casalinga.
+* Progettiamo le nostre reti per essere aperte e pubbliche: chiunque può essere operatore di un nodo e pertanto estendere la rete.
+* La nostra rete è accessibile anonimamente, né gli utenti né gli operatori dei nodi sono costretti a registrarsi per partecipare.
+* La nostra rete è non commerciale.
+* La nostra rete non è censurata.
+* Rispettiamo la legge sulla segretezza delle telecomunicazioni.
+* Protezione della privacy e minimizzazione della raccolta di dati: non teniamo registri delle connessioni e non cataloghiamo dati. Non conserviamo dati che consentono l'identificazione della persona. La divulgazione di informazioni di contatto o di coordinate geografiche sulle mappe sono opzionali.
+* Gli utenti sono responsabili per la crittografia end-to-end e per l'anonimizzazione del proprio traffico.
+
+Quando progettiamo le nostre reti facciamo attenzione al livello di decentralizzazione. Pertanto una situazione dove piccolo gruppo di amministratori o singoli individui che detengono il controllo di un'intera (sotto)rete non riflette il nostro modo di pensare. Gli operatori dei nodi hanno libera scelta sul consentire o meno l'amministrazione da remoto. Gli interventi sui nodi, ad esempio aggiornamenti del firmware o altre operazioni di amministrazione remota, devono essere eseguiti con l'esplicito consenso dei rispettivi operatori. Gli operatori dei nodi possono inoltre decidere se preferiscono fornire accesso a Internet, che sia per i partecipanti della rete mesh o per il pubblico. Quando si decide di farlo, è importante che Freifunk non venga percepita come un "fornitore di accesso ad internet gratuito". Le decisioni sui rilasci dei software all'interno delle comunità locali vengono prese assieme. Il firmware per i nostri nodi è basato su software libero. Pubblichiamo ulteriore software sviluppato sempre come FLOSS. Puntiamo a produrre documentazione di qualità che consenta agli operatori di estendere il firmware o suggerire miglioramenti. L'apertura crea fiducia nel software.
```

Comment:
_Pertanto una situazione dove piccolo gruppo di amministratori o singoli individui che detengono il controllo di un'intera (sotto)rete non riflette il nostro modo di pensare._
_**Quindi una situazione in cui poche persone possano amministrare e controllare un'intera (sotto)rete non riflette il nostro modo di pensare**_